code/modules/antagonists/blob/blob.dm

```diff
@@ -46,16 +49,52 @@
 
 	get_statistics()
 		var/list/absorbed_lifeforms = list()
+		var/list/upgrades = list()
 		for (var/mob/living/carbon/human/H in src.absorbed_victims)
 			if(!H.last_client?.key)
 				absorbed_lifeforms += "[H.real_name] (NPC)"
 
 			else
 				absorbed_lifeforms += "[H.real_name] (played by [H.last_client?.key])"
 
+		for (var/datum/blob_upgrade/upgrade in purchased_upgrades)
```

Comment:
Could maybe go through the list of upgrades and combine the repeated ones by counting the number of times they appear and putting a `x2` at the end of the displayed name?

Reply:
![blob x2](https://github.com/goonstation/goonstation/assets/48467372/ba83acb5-26ad-455c-a3db-7e9f80dfb341)
looks like this now! whahoo